### PR TITLE
Make split() a method of the manifest classes (and add CutSet support)

### PR DESCRIPTION
**Pull request by pzelasko**
Mostly a discoverability improvement + fixing lack of support for CutSet. I found the option in PyCharm to sort imports alphabetically, too...

### CODE REVIEW

**Comment by danpovey** [comment]
Looks good!  Sorry I started looking at it earlier but got interrupted and didnt get back to it.

**Comment by pzelasko** [comment]
Haha, it's my bad - these itchy fingers love those shiny merge buttons...

File: lhotse/audio.py [hunk 1]
```diff
@@ -218,6 +219,19 @@ def filter(self, predicate: Callable[[Recording], bool]) -> 'RecordingSet':
         """
         return RecordingSet.from_recordings(rec for rec in self if predicate(rec))
 
+    def split(self, num_splits: int, randomize: bool = False) -> List['RecordingSet']:
+        """
+        Split the ``RecordingSet`` into ``num_splits`` pieces of equal size.
```
Comment (csukuangfj) [inline lhotse/audio.py:224]: > into ``num_splits`` pieces of equal size

The comment is misleading and does not match the implementation.

It does not describe how it handles the case when `len(self) % num_splits != 0`. 

There is an option `drop_last` in PyTorch's dataloader and `drop_remainder` in Tensorflow's dataset.

The current implementation simply uses the semantic `drop_last=False` without
documenting it.

Comment (pzelasko) [reply lhotse/audio.py:224]: I’ll fix it (soon)

**Pull request merged**